### PR TITLE
Updated event-triggers.md

### PR DESCRIPTION
**Pull request by akramkazmi71**
### Description

Fixed some small typos and few grammatical mistakes.

What component does this PR affect? 

- [ ] Server
- [ ] Console
- [ ] CLI
- [x] Docs
- [ ] Community Content
- [ ] Build System

Requires changes from other components? If yes, please mark the components:

- [ ] Server
- [ ] Console
- [ ] CLI
- [ ] Docs
- [ ] Community Content
- [ ] Build System

### Related Issue

 [745](https://github.com/hasura/graphql-engine/issues/745)

### Type

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Docs update
- [ ] Community content

### Checklist:

- [ ] I have read the **[contributing guide](https://github.com/hasura/graphql-engine/blob/master/CONTRIBUTING.md)** and my code conforms to the guidelines.
- [ ] This change requires a change in the documentation. 
- [ ] I have updated the documentation accordingly.
- [ ] I have added required tests.

### CODE REVIEW

File: event-triggers.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ Trigger webhooks on database events using Hasura GraphQL Engine's event triggers
 
 * **Build reactive & async apps/features**: Trigger asynchronous serverless or cloud functions to reduce infrastructure costs and simplify DevOps for developers.
 
-* **Atomic & Reliable**: Using native Postgres triggers, every relevant action on the database is captured as an event. Even if Hasura is down or being updated, events will be captured and delivered as soon as possible with an *atleast-one* guarantee. You can even configure delivery policies like `max_retries` and `retry_interval`.
+* **Atomic & Reliable**: Using native Postgres triggers, every relevant action on the database is captured as an event. Even if Hasura is down or being updated, events will be captured and delivered as soon as possible with *atleast-one* guarantee. You can even configure delivery policies like `max_retries` and `retry_interval`.
```
Comment (tirumaraiselvan) [inline event-triggers.md]: The correct wording should be `with an *atleast-once* guarantee`.

File: event-triggers.md [hunk 2]
```diff
@@ -91,10 +91,10 @@ Transform and load data into external data-stores. Check out this demo and tutor
 * [Try it out](https://shahidh.in/hasura-serverless-etl)
 * [Tutorial](community/examples/serverless-etl)
 
-### Building reactive UX for your async backend with realtime GraphQL
+### Building reactive UX for your asynchronous backend with realtime GraphQL
```
Comment (tirumaraiselvan) [inline event-triggers.md]: async is more common and easier to digest

File: event-triggers.md [hunk 3]
```diff
@@ -91,10 +91,10 @@ Transform and load data into external data-stores. Check out this demo and tutor
...
 
 Propagate event-driven and asynchronous information to UI clients easily with GraphQL subscriptions & live-queries.
 
 ![Reactive apps architecture](assets/reactive-apps-arch.png)
 
-**Watch**: [Building a reactive app with an async backend](https://youtu.be/kTSOxRrtCeI) (*04:15 mins*)
+**Watch**: [Building a reactive app with an asynchronous backend](https://youtu.be/kTSOxRrtCeI) (*04:15 mins*)
```
Comment (tirumaraiselvan) [inline event-triggers.md]: async is more common and easier to digest

File: event-triggers.md (hunk 1 above)
Comment (akramkazmi71) [reply event-triggers.md]: Then I would make the changes as specified.

**Pull request closed without merging**